extensions/wikia/Oasis/Oasis.i18n.php

```diff
@@ -161,12 +161,19 @@
 your email preferences here: http://community.wikia.com/Special:Preferences',
 	'oasis-label-wiki-vertical-id-0' => 'Other',
 	'oasis-label-wiki-vertical-id-1' => 'TV',
-	'oasis-label-wiki-vertical-id-2' => 'Video Games',
```

Comment:
Did we decide to change this hub category to "Games"? If so, it looks like all the other translations for this message are translating the term "Video Games", those should be changed as well.